src/components/Confirm/index.js

```diff
@@ -163,6 +163,8 @@ class Confirm extends Component  {
   }
 
   onApiSuccess = (apiResponse) => {
+    const duration = Date.now() - this.startTime
```

Comment:
so why not track the spinner instead of the api calls?

Reply:
We use the spinner in several places so we would need to pass the action (e.g. document upload) to the spinner to track uploads reliably.

I could pull out the tracking and logic display the spinner so they are explicitly coupled, within the `Confirm` component, but that feels like overkill to me. What do you think?

Reply:
An event gets its `step` prepended if you call `this.trackScreen`